src/model/resources/storage/storage_all.jl

```diff
@@ -89,14 +111,39 @@ function storage_all!(EP::Model, inputs::Dict, setup::Dict)
 		# Maximum discharging rate must be less than power rating OR available stored energy in the prior period, whichever is less
 		# wrapping from end of sample period to start of sample period for energy capacity constraint
 		@constraints(EP, begin
-			[y in STOR_ALL, t=1:T], EP[:vP][y,t] <= EP[:eTotalCap][y]
-			[y in STOR_ALL, t=1:T], EP[:vP][y,t] <= EP[:vS][y, hoursbefore(hours_per_subperiod,t,1)]*dfGen[y,:Eff_Down]
+			[y in STOR_ALL, t=1:T], EP[:vP][y,t] + EP[:vCAPRES_discharge][y,t] <= EP[:eTotalCap][y]
+			[y in STOR_ALL, t=1:T], EP[:vP][y,t] + EP[:vCAPRES_discharge][y,t] <= EP[:vS][y, hoursbefore(hours_per_subperiod,t,1)]*dfGen[y,:Eff_Down]
 		end)
 	end
 	#From co2 Policy module
 	@expression(EP, eELOSSByZone[z=1:Z],
 		sum(EP[:eELOSS][y] for y in intersect(STOR_ALL, dfGen[dfGen[!,:Zone].==z,:R_ID]))
 	)
+
+	# Capacity Reserve Margin policy
+	if capacity_reserve_margin == 1
```

Comment:
`storage.jl` has `if CapacityReserveMargin > 0` rather than if it `== 1`. Should we plan for a future with multiple levels (of intensity? sophistication?) of capacity reserve margin policies?

Note to us that at some point we should coordinate these statements (or maybe make policies booleans!)